docker-compose.yml

```diff
@@ -53,7 +53,7 @@ services:
         ports:
             - '${FORWARD_DB_PORT:-3306}:3306'
         volumes:
-            - 'mariadb-data:/var/lib/mysql'
+            - 'mariadb-data_prod_05192024:/var/lib/mysql'
```

Comment:
OMG! I restarted my machine with this branch checked out and thought my DB got nuked. So I rebuilt my VM and it happened again. Then I spent like two hours trying to figure out how to get to the data that was clearly still taking up space on my HD. I finally saw the "data_prod_05192024" in one of the messages and grep'd to find this.

Reply:
😓 I am so sorry for that. I really need to get a handle on gitignoring changes that don't belong so I'm not constantly committing/not committing stuff that doesn't belong.

I've fixed the branch, and I commit to you I'll get a better handle on this.